types/src/system/auction/unbonding_purse.rs

```diff
@@ -140,8 +139,8 @@ mod tests {
     };
 
     const BONDING_PURSE: URef = URef::new([41; 32], AccessRights::READ_ADD_WRITE);
-    const ERA_OF_WITHDRAWAL: EraId = EraId::max_value();
 
+    static ERA_OF_WITHDRAWAL: Lazy<EraId> = Lazy::new(|| EraId::from(u64::max_value()));
```

Comment:
If you use `EraId::from(u64::max_value())` often, you could add an `EraId::MAX` constant.